src/style.css

```diff
@@ -1,3 +1,74 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  align-items: center;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+ul {
+  list-style: none;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+li {
+  margin: 0 10px;
+  display: flex;
+}
+
+li a {
+  display: flex;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for your links instead